git/src/providers/utils.js

```diff
@@ -1,5 +1,12 @@
 // eslint-disable-next-line no-magic-numbers
 const MILLISECONDS_IN_HOURS = 1000 * 3600;
+const SEVEN_DAYS = 7;
```

Comment:
![image](https://user-images.githubusercontent.com/5349650/127056373-a1bd3784-8968-41a2-ab32-c26e814db55b.png)

BTW, weren't we going to do it for 14 days (a sprint)?